src/tool_help.c

```diff
@@ -215,6 +215,9 @@ void tool_version_info(void)
     printf("Features:");
     for(builtin = feature_names; *builtin; ++builtin)
       printf(" %s", *builtin);
+#ifdef CURL_CA_EMBED
+    printf(" CAcert");
+#endif
```

Comment:
Is there a succint way to print it at its sorted position?

This is almost there except one corner case where it's shown first:
```c
    for(builtin = feature_names; *builtin; ++builtin) {
#ifdef CURL_CA_EMBED
      const char *ca_embed_feat = "CAcert";
      if(struplocompare(ca_embed_feat, *builtin) < 0 || !*(builtin+1)) {
        printf(" %s", ca_embed_feat);
      }
#endif
      printf(" %s", *builtin);
    }
```

Reply:
What corner case?

If that that can't be fixed easy enough, I suppose simply qsorting the list is another option.

Reply:
If we ever want to add a second feature in the tool to that list we would probably prefer to sort it anyway.

Reply:
The corner case is if there is a single feature in the list and that's starting with an A or B. Output ending up as `CAcert alt-svc` for example.